test/CMakeLists.txt

```diff
@@ -67,7 +67,14 @@ macro(onedpl_add_test test_source_file switch_off_checked_iterators)
     get_filename_component(_test_name ${test_source_file} NAME)
     string(REPLACE "\.cpp" "" _test_name ${_test_name})
 
-    add_executable(${_test_name} EXCLUDE_FROM_ALL "${test_source_file}")
+    # add additional translation units to current test according to mask
+    get_filename_component(_test_path ${test_source_file} DIRECTORY)
+    string(REPLACE "\.pass" "" _test_name_without_pass ${_test_name})
+    file(GLOB_RECURSE additional_src_files "${_test_path}/${_test_name_without_pass}_*.cpp")
```

Comment:
I think `GLOB`  with `${_test_path}` specified as the directory seems to be the intention here instead of `GLOB_RECURSE`, which searches all subdirectories.

Reply:
Fixed.